commit 0a9cc19183cd0d542b58271c6f3cf78b93b15e40 (refs/changes/86/786/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-26T12:11:32-07:00 (2 years, 11 months ago)
    
    IMGAPI-599 IMGAPI client from node-sdc-clients at version 10.0.2 is not usable (restify/clients#90 fix)

diff --git a/CHANGES.md b/CHANGES.md
index 216ec15..691c97a 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -12,6 +12,11 @@
 
 ## not yet released
 
+## 10.0.3
+
+- IMGAPI-599 restify/clients#90 fix '<jsonclient>.post(...)' without a body
+  argument
+
 ## 10.0.2
 
 - TOOLS-1584 Ensure 'filters.channel' passed to IMGAPI.listImages wins over a
diff --git a/package.json b/package.json
index f93cf6d..2e118ae 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-clients",
   "description": "node.js client libraries for Triton core REST APIs.",
-  "version": "10.0.2",
+  "version": "10.0.3",
   "homepage": "http://www.joyent.com",
   "repository": {
     "type": "git",
@@ -17,7 +17,7 @@
     "lru-cache": "2.3.0",
     "libuuid": "0.2.1",
     "once": "^1.3.1",
-    "restify-clients": "^1.4.0",
+    "restify-clients": "^1.4.1",
     "restify-errors": "^3.0.0",
     "vasync": "^1.6.2",
     "verror": "^1.6.0",
